docs/src/all_workflows_parameters.md

```diff
@@ -0,0 +1,76 @@
+# Index of Workflows Parameters
+
+List of all workflows parameters. For more information on them please visit the dedicated user guides.
+
+## Required Parameters
+
+These must be provided, or the workflow will not run.
+
+- `ESTIMANDS_CONFIG`: YAML configuration file describing the effect sizes of interest.
+- `TRAITS_DATASET`: Path to a traits dataset. If you are running this for a non-UKB cohort, your sample IDs must be specified in the first column of this CSV file, with the column name `SAMPLE_ID`.
+- `BED_FILES`: Path to PLINK BED files.
+- `BGEN_FILES`: Path to imputed BGEN files (optional for a GWAS).
```

Comment:
Maybe specify that BGEN files must be indexed with bgenix prior to use of the targene-pipeline, as this is not always the case.